src/Rive.tsx

```diff
@@ -316,7 +357,16 @@ const RiveContainer = React.forwardRef<RiveRef, Props>(
           stateMachineName={stateMachineName}
         />
 
-        <View style={styles.children}>{children}</View>
+        <TouchableWithoutFeedback
```

Comment:
We should wrap this component around `RiveViewManager` as this should be the component actually rendering the Rive animations. The children View it's currently wrapped around won't be of interest since its just content consumers can add to `<Rive>{some-children-nodes}</Rive>`

The other thing is - we should move `<View>{children}</View>` above `RiveViewManager` so that this `position: absolute` component isn't overlaying and blocking the Touchable component.